__tests__/index.js

```diff
@@ -0,0 +1,21 @@
+import request from 'supertest';
```

Comment:
Is supertest better to use?

Reply:
Not that it was better. I used it because I saw it in their documentation and other examples I saw used supertest. http://www.albertgao.xyz/2017/05/24/how-to-test-expressjs-with-jest-and-supertest/